### PR TITLE
Add urijs template expand instead of own build endpoint logic, tests

### PR DESCRIPTION
**Pull request by vedrani**

### CODE REVIEW

File: test/actions/create.spec.js [hunk 1]
```diff
@@ -202,7 +202,7 @@ describe('Create action creator', () => {
       headers: {
         'Content-Type': 'application/vnd.api+json',
       },
-      endpoint: 'api.test/{param1}/{param2}/abc',
+      endpoint: 'api.test/{param1}/{param2}/abc{?q1,q2}',
```
Comment (zrumenjak) [inline test/actions/create.spec.js]: Why did you add this q1,q2?

File: test/schemaConfig.spec.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-unused-expressions */
+import { expect } from 'chai';
+import {
+  buildEndpoint,
+} from '../src/schemaConfig';
+
+describe('Build endpoint', () => {
+  it('static', () => {
+    const endpoint = 'http://new.shoutem.com';
+    const params = { test: 1};
```
Comment (zrumenjak) [inline test/schemaConfig.spec.js]: Add a space after 1 :)

Comment (zrumenjak) [inline test/schemaConfig.spec.js]: In all tests...

File: test/actions/create.spec.js (hunk 1 above)
Comment (vedrani) [inline test/actions/create.spec.js]: Idea was to test will urijs resolve query parameters that are passed as `q1`, `q2`
